experiments/spotify/5_develop_contrib_source_from_scratch/README.md

```diff
@@ -0,0 +1,44 @@
+# Developing a contrib source from scratch
+
+We develop new source with the intention to be importable. Probably we are a software developer or an agency and our source is something really popular ie. Hubspot. The workflow is not that different from `develop contrib source from own source` but:
+
+1. The source structure would be much cleaner and decorators will be used, not `as_table`
```

Comment:
I don't understand why we don't just use decorators in 4. Can you help me understand?

Reply:
again I do not agree IMO decorators are a part of every on-framework - they let you keep your current code but us it in a new way. ie caching decorators are quite popular because
1. they hide a lot of complexity for the user
2. give the developer huge amount of control over the decorated code!